docs/actions/initialize-form.md

```diff
@@ -0,0 +1,65 @@
+  
+## `Related Actions`
+
+| Action Name                                       | Description                                                                                       |
+| ------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| [Open Form Popup](/docs/actions/open-form-popup)       | Opens a a specified Form in a pop-up.|
+
+## `Input Parameter Reference`
+
+| Parameter                  | Description                                                                                                                                                                | Supports Tokens | Default            | Required |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | ------------------ | -------- |
+| Description            | A short description for the action. Only Citizen Developers and Admins will be able to see this field. | No              | `empty string` | No      |
+| Error Message                  | A generic error message that will be displayed to end users in case an error occurs in the action.  Citizen Developer and above will also see the detailed error message provided by the system.  Leave empty to use the default message.                                                                                                 | Yes             | `empty string`     | No      |
+| Condition                   | This boolean expression is used to determine if this action will execute. Use it to enable or disable this action programatically. For example, you’d enable a ShowError action only if you’ve found an error let’s say when you parsed a response from a web service. A common example is [SomeField] == “Some Value”.                                                                 | Yes             | `empty string`     | No       |
+| Select Module | Select the Form module that you wish to initializeby selecting it from a prepopulated list of all the Forms on the page. Alternativly you can set it programatically by switching to an expression to use My Tokens or values based on other fields.                                                                                                                      | Yes              | `None selected`    | Yes      |
+| QueryString Parameters                 | Optionally enter parameters as Name/Value pairs that can be accessed through querystring. The values set here will be sent to the Form module and will be available in the Init and PreInit actions as values of the 'QueryString:' token namespace.                                                                                                                                    | Yes              | `None`               | No       |
+|
+
+## `Output Parameters Reference`
```

Comment:
I have been leaving this section out if there aren't any output params. Do you think we should always have it in place?

Reply:
We can leave it out.  Thought it would be good to keep a consistent structure and implicitly state that it doesn't apply so people know there is no output token.  In this case, the consistent structure would be to leave it out since that's what we're already doing :-)